src/templates/next/components/internal/Notification/Notification.tsx

```diff
@@ -14,7 +14,10 @@ const NotificationBanner = ({ content }: NotificationProps) => {
   return (
     isShown && (
       <div className="flex flex-col sm:flex-row justify-between gap-6 bg-site-primary-100 py-4 px-5 sm:px-10">
-        <BaseParagraph className="text-base" content={content}></BaseParagraph>
+        <BaseParagraph
+          className="text-base text-content"
+          content={content}
+        ></BaseParagraph>
```

Comment:
Assuming that this was intended to go in for a different issue!

Reply:
Oh yeah this one was a separate fix for the paragraph one, missed this one when splitting up the original PR.